### ISSUE TITLE
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './jsx-runtime' is not defined by "exports" in package.json

### ISSUE THREAD

**Issue by berndartmueller** [yhatt/jsx-slack#231]
Hey everyone!

I've just upgraded from 4.0.0 to 4.2.0 and now I'm getting the following error:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './jsx-runtime' is not defined by "exports" in /.../node_modules/jsx-slack/package.json
    at new NodeError (node:internal/errors:329:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:337:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:562:3)
    at resolveExports (node:internal/modules/cjs/loader:473:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:513:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:910:27)
    at Function.Module._resolveFilename (/.../node_modules/module-alias/index.js:49:29)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/.../blocks.js:15:21)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

Thanks a lot!

**Comment by yhatt** [comment]
Thanks for feedback! It brings by lack of ES modules support for JSX runtime. I have delayed to work this because React is not supported ES modules officially, but should consider to support ES modules also in `jsx-runtime.js`.

I'll do it right away, but you can use v4.1.0 instead if you're in a hurry.

**Comment by yhatt** [comment]
Fixed in the latest release [v4.2.1](https://github.com/yhatt/jsx-slack/releases/tag/v4.2.1). Please try upgrading again!

**Comment by berndartmueller** [comment]
Awesome!! Works great!

Thanks a lot, @yhatt 

I have experienced many similar issues due to the gradual switch to ES modules in open source JS. This will probably cause us many more headaches in the near future :D
